docs/src/description/appendix.md

```diff
@@ -0,0 +1,7 @@
+# Appendix
+
+Contains additional information about some concepts that are not covered in the main documentation.
+
+## Main v/s Aux Execution Trace
+
+With Randomized AIR, construction of the execution trace is split into multiple stages. During the first stage, the *main trace segment* is built in a manner similar to how the trace is built for regular AIR. In the subsequent stages, *auxiliary trace segments* are built. When building auxiliary trace segments, the prover has access to extra randomness sent by the verifier (in the non-interactive version of the protocol, this randomness is derived from the previous trace segment commitments). In winterfell, the number of auxiliary trace segments is limited to one.
```

Comment:
I would make this header "## Main vs. auxiliary execution trace segments (`main` and `aux`)"

I would also adjust the explanation a bit, maybe as follows (but we should get a read from @Al-Kindi-0 

With Randomized AIRs, the construction of the execution trace can split into multiple rounds, with the verifier providing new randomness between rounds. In the first round, before any random values are available, the Prover builds the first segment of the execution trace, which we refer to as the `main` trace (using Winterfell terminology). Additional trace segments can be built after the prover has committed to the first execution trace segment.  When building auxiliary trace segments, the prover has access to extra randomness sent by the verifier (in the non-interactive version of the protocol, this randomness is derived from the previous trace segment commitments). We refer to these as auxiliary trace segments (using Winterfell terminology again), and we call the the first one the `aux` segment in AirScript and throughout this documentation.

Miden VM only makes use of one auxiliary trace segment, and in Winterfell the number of auxiliary trace segments is currently limited to one. Therefore, AirScript currently only supports the main execution trace `main` and a single auxiliary trace segment called `aux`.